modules.d/40network/ifup.sh

```diff
@@ -416,6 +417,9 @@ for p in $(getargs ip=); do
                 do_dhcp -6 ;;
             auto6)
                 do_ipv6auto ;;
+            auto6-dhcp6)
+                ret=do_ipv6auto ;;
+                [ $ret -ne 0 ] && do_dhcp -6
```

Comment:
1. ```;;```are in the first line, so the switch stops early
2. ret=... does not assign the exit value of the function to a variable


This should be:
```
do_ipv6auto || do_dhcp -6 ;;
```

Reply:
Yes, will fix it